test/Performance/benchmark/sqlsrv/regular/SqlsrvCreateDbTableProcBench.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use SqlsrvPerfTest\SqlsrvUtil;
+/**
+ * @BeforeMethods({"connect"})
+ * @AfterMethods({"disconnect"})
+ */
+class SqlsrvCreateDbTableProcBench{
+    private $conn;
+
+    public function connect(){
+        $this->conn = SqlsrvUtil::connect();
+    }
+
+    /*
+    * Each iteration creates a database, a table and a stored procedure in that database and drops the database at the end.
+    * Not that, execDirect function are used to execute all the queries. 
+    */
+    public function benchCreateDbTableProc(){
+        $randomNum = rand();
+        $databaseName = "test_db_$randomNum";
+        $tableName = "test_table_$randomNum";
+        $procName = "test_proc_$randomNum";
+        SqlsrvUtil::createDbTableProc( $this->conn, $databaseName, $tableName, $procName );
```

Comment:
I mean why the function should be in the class at all, if it's only used here, and not gonna be used anywhere else?